Formula/trino.rb

```diff
@@ -37,18 +37,6 @@ def install
     libexec.mkpath
     system "tar", "-C", libexec.to_s, "--strip-components", "1", "-xzf", "trino-server-#{version}.tar.gz"
 
-    # Manually untar, since macOS-bundled tar produces the error:
-    #   trino-363/plugin/trino-hive/src/test/resources/<truncated>.snappy.orc.crc: Failed to restore metadata
-    # Remove when https://github.com/trinodb/trino/issues/8877 is fixed
-    resource("trino-src").stage do |r|
-      ENV.prepend_path "PATH", Formula["gnu-tar"].opt_libexec/"gnubin"
-      system "tar", "-xzf", "trino-#{r.version}.tar.gz"
-      (libexec/"etc").install Dir["trino-#{r.version}/core/docker/default/etc/*"]
-      inreplace libexec/"etc/node.properties", "docker", tap.user.downcase
-      inreplace libexec/"etc/node.properties", "/data/trino", var/"trino/data"
-      inreplace libexec/"etc/jvm.config", %r{^-agentpath:/usr/lib/trino/bin/libjvmkill.so$\n}, ""
-    end
-
```

Comment:
The upstream issue ticket says that you should use zip archive instead of tar.gz.
So we should use .zip archive instead of removing this section.
I created a rough patch for this. How about it?
https://gist.github.com/darklore/63c30a81be14ce811ec82afdc6eb56b5